src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/CollectionRuleOptionsTests.cs

```diff
@@ -249,6 +249,301 @@ public Task CollectionRuleOptions_EventCounterTrigger_GreaterThanLargerThanLessT
                 });
         }
 
+        [Fact]
+        public Task CollectionRuleOptions_AspNetRequestCountTrigger_MinimumOptions()
+        {
+            const int ExpectedRequestCount = 10;
+
+            return ValidateSuccess(
+                rootOptions =>
+                {
+                    rootOptions.CreateCollectionRule(DefaultRuleName)
+                        .SetAspNetRequestCountTrigger(options =>
+                        {
+                            options.RequestCount = ExpectedRequestCount;
+                        });
+                },
+                ruleOptions =>
+                {
+                    AspNetRequestCountOptions requestCountOptions = ruleOptions.VerifyAspNetRequestCountTrigger();
+                    Assert.Equal(ExpectedRequestCount, requestCountOptions.RequestCount);
+                });
+        }
+
+        [Fact]
+        public Task CollectionRuleOptions_AspNetRequestCountTrigger_RoundTrip()
+        {
+            const int ExpectedRequestCount = 10;
+            TimeSpan ExpectedSlidingWindowDuration = TimeSpan.FromSeconds(45);
+            string[] ExpectedIncludePaths = { "IncludePath1", "IncludePath2" };
```

Comment:
Not something we need to worry about for this PR, but I am wondering if we should also bring in the custom validator for these properties from the diag repo.